spec/controllers/user/teams_controller_spec.rb

```diff
@@ -0,0 +1,63 @@
+require 'rails_helper'
+require 'user/teams_controller'
```

Comment:
You can remove this line if you want, `require 'rails_helper'` will already setup the paths and `User::TeamsController` will be available via autoload.